pkgs/development/libraries/science/math/cliquer/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.21";
+  name = "cliquer-${version}";
+
+  # autotoolized version of the original cliquer
```

Comment:
I don't think this package should have the name cliquer then. And the homepage should also be changed accordingly.

Reply:
Its still the same package with the same functionality, just an improved / updated build. I think thats an implementation detail that would only confuse users that want to install cliquer.

However if you prefer a name change and have a different name in mind, I'm not strongly opposed.

Reply:
Considering that Arch is also using autocliquer, this seems alright after all: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/cliquer#n15
However please add a `downloadPage` pointing to autocliquer.

Reply:
You could use
```
downloadPage = src.meta.homepage;
```